bin/copyRawDataToPrm.sh

```diff
@@ -554,7 +554,8 @@ else
 		#
 		filePrefix="$(basename "${sampleSheet%."${SAMPLESHEET_EXT}"}")"
 		controlFileBase="${PRM_ROOT_DIR}/logs/${filePrefix}/"
-		export JOB_CONTROLE_FILE_BASE="${controlFileBase}/${filePrefix}.${SCRIPT_NAME}"
+		runPrefix="run01"
```

Comment:
Rawdata heeft geen run01, toch? Dat is alleen de NGS_DNA pipeline, die een run01 maakt...